foundations/javascript_basics/fundamentals-1.md

```diff
@@ -70,6 +70,7 @@ Numbers are the building blocks of programming logic!  In fact, it's hard to thi
 
 ### Assignment
```

Comment:
The heading "Practice" feels more aligned with the content in this section as opposed to "Assignment". Can you change this as well?

Reply:
I had that in mind, However in the other fundementals the "Practice" has no container background like the Assignment sections. Would you have me change all the Practices to have a background or to change THIS practice and remove it's background to be more alligned with the rest?

Reply:
Thanks for asking. Practice sections did not use to have a div container. But with the new [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md), Practice should have one.

We're aware that some lessons have inconsistency in Practice sections as well as others. A curriculum wide project will deal with them soon. So for now, you don't need to change Practices in the other lessons. Just changing the heading from "Assignment" to "Practice" for this lesson should be enough.

Reply:
I tracked down the 4 other practices and added background to them. If this going to interrupt an ingoing change, and you want me to revert them back and only change “Assignment” to “Practice” I will do so. However, if what you meant is, that I don't have to, but I can if I wanted to, then I will push the changes. Sorry for the inconvenience waiting an answer.

Reply:
Yes, please revert the other changes and only push the requested change for this PR. 

And it's not an inconvenience, I appreciate you asking! :smile: